plugins/tuya/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
not sure if this file is actually used, the scrypted plugin build process just assumes a specific file structure and has a tsconfig.json under the hood.

Reply:
cool, Ill remove it